frontend/src/lib/components/store.js

```diff
@@ -0,0 +1,3 @@
+import { writable } from 'svelte/store';
+
+export const selected_store = writable([]);
```

Comment:
Can you create a folder called 'stores", put this file into there and rename it to `store.ts`? `components` should contain only the svelte components.